sentry_sdk/integrations/opentelemetry/sampler.py

```diff
@@ -45,6 +45,18 @@ def dropped_result(span_context):
     # type: (SpanContext) -> SamplingResult
     trace_state = span_context.trace_state.update(SENTRY_TRACE_STATE_DROPPED, "true")
 
+    # Tell Sentry why we dropped the transaction/span
```

Comment:
to match the logic in the earlier tracing impl, we should rather do this at the end of the transaction flow, so I'd not do this here but in span processor `on_end` and only for the root span case. Here it will apply to every span and the numbers will be wrong.